docs/framework/guides/deep-dive/localization.md

```diff
@@ -8,7 +8,7 @@ category: framework-deep-dive
 
 All CKEditor 5 WYSIWYG editor features support message localization. It means that the user interface of any feature can be translated into various languages and regions depending on the user's preferences.
 
-Starting with the release of CKEditor 5 v19.0.0 as well as CKEditor 5 development tools we opened the translation system to third-party plugins. We also provided a way for adding missing or fixing invalid translations to the editor and introduced support for translating plural forms.
+CKEditor 5 translation system is opened to third-party plugins. Any custom features that you might introduce can be localized. We also provided a way for adding missing or overwriting existing translations and we support translating plural forms.
 
 <info-box warning>
 	Make sure to use up-to-date CKEditor 5 development tool packages. Older versions of the tools do not provide support for features described in this guide.
```

Comment:
We removed the version reference from the sentence above, so maybe it makes sense to mention it now.

```suggestion
	Make sure to use up-to-date CKEditor 5 development tool packages. Versions of the tools older than v19.0.1 do not provide support for features described in this guide.
```